dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/ProcessInstanceMapMapper.xml

```diff
@@ -19,7 +19,7 @@
 <!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN" "http://mybatis.org/dtd/mybatis-3-mapper.dtd" >
 <mapper namespace="org.apache.dolphinscheduler.dao.mapper.ProcessInstanceMapMapper">
     <sql id="baseSql">
-        id, parent_process_instance_id, parent_task_instance_id, process_instance_id
+        id, parent_process_instance_id, parent_task_instance_id, process_instance_id,next_process_instance_id
```

Comment:
There is no column `next_process_instance_id` in schema `t_ds_relation_process_instance`, this will throw exception at runtime

Reply:
does this fixed?